SDKBuildScripts/Update-PlayFabSdk.ps1

```diff
@@ -54,6 +54,15 @@ Indicites whether or not to remove all generated source files from the destinati
 .PARAMETER Beta
 Indicates whether or not to include any APIs tagged with as beta.
 
+.PARAMETER BuildFlags
```

Comment:
As you're the primary user and more-or-less owner of this file, I'll approve any changes you want to make to this file, if they're made in isolation.
I can't approve this PR with the other changes though.
In particular, we should never make a PR with changes to multiple SDKs at once.

Reply:
Oh wow, I missed a major problem with this PR.
You can't change the generate.js file directly, as it's an intermediate file generated from generate.ts.
You have to change the generate.ts file, which is the "source" file, and then commit both.
ALSO, that would have to be in a yet third PR.
As for the changes in generate.[ts], you are perhaps mis-understanding the purpose of genConfig.json.
Beta should not be generated to CSharpSDK at all, and we enforce the genConfig.json above the command line inputs on purpose.
If you want CSharp with Beta, you should be building to:
https://github.com/PlayFab/CSharpBetaSdk